### ISSUE TITLE
No bounding boxes

### ISSUE THREAD

**Issue by codinalot** [tensorflow/models#2912]
------------------------

### System information
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.4
- **Bazel version (if compiling from source)**: n/a
- **CUDA/cuDNN version**: CUDA v8.0
- **GPU model and memory**: GTX 1070 8.00GiB
- **Exact command to reproduce**: train ssd_mobilenet_v1 with 1 class, run eval

### Describe the problem

I am attempting to train a novel dataset object detector. I have tried many different sample config files. For simplicity, this issue happens with ssd_mobilenet_v1_pet.config found in the sample configs. I modify the file to have num_classes = 1. I train, and it has no issues. However, when I go to evaluate (which I cannot run at the same time due to weird NVIDIA STATUS issues other people are also having), on tensorboard the images contain no bounding boxes whatsoever; even for epoch 0. Also, the only images I see are labeled "image-#" and don't contain any bounding boxes. This is odd, because the loss drops as the training runs. I'm not sure what could be going wrong here, as I expect it would tell me if I had an issue with my input file format.

### Source code / logs
--output from running eval.py:
`INFO:tensorflow:Restoring parameters from Resized/models/model/train\model.ckpt-1726
INFO:tensorflow:Restoring parameters from Resized/models/model/train\model.ckpt-1726
WARNING:root:image 0 does not have groundtruth difficult flag specified
WARNING:root:image 1000 does not have groundtruth difficult flag specified`

--output from running train.py
`C:\Users\mccla\Desktop\School\DeepLearning\TensorflowModels\research>python object_detection/train.py --logtostderr --pipeline_config_path="Resized/models/model/ssd_mobilenet_v1_pets.config" --train_dir="Resized/models/model/train"
WARNING:tensorflow:From C:\Users\mccla\Desktop\School\DeepLearning\TensorflowModels\research\object_detection\trainer.py:210: create_global_step (from tensorflow.contrib.framework.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.create_global_step
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:Summary name /clone_loss is illegal; using clone_loss instead.
2017-11-27 13:44:24.339030: I C:\tf_jenkins\home\workspace\rel-win\M\windows-gpu\PY\36\tensorflow\core\platform\cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2017-11-27 13:44:24.618270: I C:\tf_jenkins\home\workspace\rel-win\M\windows-gpu\PY\36\tensorflow\core\common_runtime\gpu\gpu_device.cc:1030] Found device 0 with properties:
name: GeForce GTX 1070 major: 6 minor: 1 memoryClockRate(GHz): 1.683
pciBusID: 0000:01:00.0
totalMemory: 8.00GiB freeMemory: 6.63GiB
2017-11-27 13:44:24.618437: I C:\tf_jenkins\home\workspace\rel-win\M\windows-gpu\PY\36\tensorflow\core\common_runtime\gpu\gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1070, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Starting Session.
INFO:tensorflow:Saving checkpoint to path Resized/models/model/train\model.ckpt
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 1: loss = 12.7232 (40.362 sec/step)
INFO:tensorflow:global step 2: loss = 11.6273 (1.335 sec/step)
INFO:tensorflow:global step 3: loss = 11.1959 (1.355 sec/step)
INFO:tensorflow:global step 4: loss = 11.2235 (1.374 sec/step)
INFO:tensorflow:global step 5: loss = 10.8122 (1.429 sec/step)
INFO:tensorflow:global step 6: loss = 10.6125 (1.436 sec/step)
INFO:tensorflow:global step 7: loss = 11.0357 (1.401 sec/step)
INFO:tensorflow:global step 8: loss = 9.8922 (1.353 sec/step)
INFO:tensorflow:global step 9: loss = 10.5882 (1.367 sec/step)
INFO:tensorflow:global step 10: loss = 10.5067 (1.377 sec/step)
INFO:tensorflow:global step 11: loss = 10.5667 (1.380 sec/step)
INFO:tensorflow:global step 12: loss = 10.5605 (1.443 sec/step)
INFO:tensorflow:global step 13: loss = 9.7699 (1.414 sec/step)
INFO:tensorflow:global step 14: loss = 9.6726 (1.383 sec/step)
INFO:tensorflow:global step 15: loss = 9.4765 (1.363 sec/step)
INFO:tensorflow:global step 16: loss = 9.9130 (1.376 sec/step)
INFO:tensorflow:global step 17: loss = 9.1789 (1.353 sec/step)
INFO:tensorflow:global step 18: loss = 9.1648 (1.425 sec/step)
INFO:tensorflow:global step 19: loss = 9.1064 (1.410 sec/step)
INFO:tensorflow:global step 20: loss = 9.4705 (1.399 sec/step)
INFO:tensorflow:global step 21: loss = 8.4385 (1.380 sec/step)
INFO:tensorflow:global step 22: loss = 8.6938 (1.359 sec/step)
INFO:tensorflow:global step 23: loss = 8.8277 (1.466 sec/step)
INFO:tensorflow:global step 24: loss = 7.9505 (1.415 sec/step)
INFO:tensorflow:global step 25: loss = 8.6651 (1.347 sec/step)
INFO:tensorflow:global step 26: loss = 9.2527 (1.388 sec/step)
INFO:tensorflow:global step 27: loss = 8.5617 (1.351 sec/step)
INFO:tensorflow:global step 28: loss = 8.1608 (1.353 sec/step)
INFO:tensorflow:global step 29: loss = 7.7482 (1.426 sec/step)
INFO:tensorflow:global step 30: loss = 8.4438 (1.442 sec/step)
INFO:tensorflow:global step 31: loss = 8.3075 (1.361 sec/step)
INFO:tensorflow:global step 32: loss = 8.4637 (1.369 sec/step)
INFO:tensorflow:global step 33: loss = 8.2178 (1.341 sec/step)
INFO:tensorflow:global step 34: loss = 7.6799 (1.332 sec/step)
INFO:tensorflow:global step 35: loss = 8.1441 (1.335 sec/step)
INFO:tensorflow:global step 36: loss = 7.8093 (1.330 sec/step)
INFO:tensorflow:global step 37: loss = 7.6860 (1.336 sec/step)
INFO:tensorflow:global step 38: loss = 8.1498 (1.352 sec/step)
INFO:tensorflow:global step 39: loss = 8.3213 (1.344 sec/step)
INFO:tensorflow:global step 40: loss = 7.7979 (1.368 sec/step)
INFO:tensorflow:global step 41: loss = 8.0037 (1.393 sec/step)
INFO:tensorflow:global step 42: loss = 7.9027 (1.394 sec/step)
INFO:tensorflow:global step 43: loss = 8.4704 (1.325 sec/step)
INFO:tensorflow:global step 44: loss = 8.3230 (1.374 sec/step)
INFO:tensorflow:global step 45: loss = 7.9647 (1.399 sec/step)
INFO:tensorflow:global step 46: loss = 8.1657 (1.360 sec/step)
INFO:tensorflow:global step 47: loss = 7.7558 (1.340 sec/step)
INFO:tensorflow:global step 48: loss = 7.9220 (1.382 sec/step)
INFO:tensorflow:global step 49: loss = 7.8658 (1.417 sec/step)
INFO:tensorflow:global step 50: loss = 7.6225 (1.357 sec/step)
INFO:tensorflow:global step 51: loss = 7.8941 (1.405 sec/step)
INFO:tensorflow:global step 52: loss = 8.3371 (1.442 sec/step)
INFO:tensorflow:global step 53: loss = 7.8570 (1.363 sec/step)
INFO:tensorflow:global step 54: loss = 7.7056 (1.359 sec/step)
INFO:tensorflow:global step 55: loss = 7.3963 (1.347 sec/step)
INFO:tensorflow:global step 56: loss = 7.6138 (1.336 sec/step)
INFO:tensorflow:global step 57: loss = 7.3227 (1.343 sec/step)
INFO:tensorflow:global step 58: loss = 7.6475 (1.325 sec/step)
INFO:tensorflow:global step 59: loss = 7.4883 (1.439 sec/step)
INFO:tensorflow:global_step/sec: 0.496778
INFO:tensorflow:Recording summary at step 59.
INFO:tensorflow:global step 60: loss = 7.3837 (2.239 sec/step)
INFO:tensorflow:global step 61: loss = 7.6624 (1.332 sec/step)
INFO:tensorflow:global step 62: loss = 7.2862 (1.348 sec/step)
INFO:tensorflow:global step 63: loss = 7.8218 (1.359 sec/step)
INFO:tensorflow:global step 64: loss = 7.4995 (1.382 sec/step)
INFO:tensorflow:global step 65: loss = 7.5226 (1.393 sec/step)
INFO:tensorflow:global step 66: loss = 7.6969 (1.371 sec/step)
INFO:tensorflow:global step 67: loss = 7.5091 (1.449 sec/step)
INFO:tensorflow:global step 68: loss = 7.6505 (1.354 sec/step)
INFO:tensorflow:global step 69: loss = 7.5332 (1.356 sec/step)
INFO:tensorflow:global step 70: loss = 7.1444 (1.448 sec/step)
INFO:tensorflow:global step 71: loss = 7.5233 (1.407 sec/step)
INFO:tensorflow:global step 72: loss = 7.2892 (1.389 sec/step)
INFO:tensorflow:global step 73: loss = 7.6981 (1.389 sec/step)
INFO:tensorflow:global step 74: loss = 8.3244 (1.419 sec/step)
INFO:tensorflow:global step 75: loss = 7.7325 (1.443 sec/step)
INFO:tensorflow:global step 76: loss = 7.5137 (1.376 sec/step)
INFO:tensorflow:global step 77: loss = 7.6614 (1.353 sec/step)
INFO:tensorflow:global step 78: loss = 8.2635 (1.357 sec/step)
INFO:tensorflow:global step 79: loss = 6.9590 (1.336 sec/step)
INFO:tensorflow:global step 80: loss = 7.3564 (1.344 sec/step)
INFO:tensorflow:global step 81: loss = 7.2654 (1.336 sec/step)
INFO:tensorflow:global step 82: loss = 7.0795 (1.348 sec/step)
INFO:tensorflow:global step 83: loss = 7.6748 (1.359 sec/step)
INFO:tensorflow:global step 84: loss = 7.1564 (1.361 sec/step)
INFO:tensorflow:global step 85: loss = 6.8588 (1.360 sec/step)
INFO:tensorflow:global step 86: loss = 7.2792 (1.375 sec/step)
INFO:tensorflow:global step 87: loss = 7.1162 (1.396 sec/step)
INFO:tensorflow:global step 88: loss = 7.4401 (1.381 sec/step)
INFO:tensorflow:global step 89: loss = 7.3341 (1.374 sec/step)
INFO:tensorflow:global step 90: loss = 7.0912 (1.353 sec/step)
INFO:tensorflow:global step 91: loss = 7.5861 (1.347 sec/step)
INFO:tensorflow:global step 92: loss = 7.5384 (1.344 sec/step)
INFO:tensorflow:global step 93: loss = 6.8495 (1.353 sec/step)
INFO:tensorflow:global step 94: loss = 7.7628 (1.355 sec/step)
INFO:tensorflow:global step 95: loss = 7.1702 (1.359 sec/step)
INFO:tensorflow:global step 96: loss = 6.8957 (1.346 sec/step)
INFO:tensorflow:global step 97: loss = 7.3544 (1.359 sec/step)
INFO:tensorflow:global step 98: loss = 7.3709 (1.340 sec/step)
INFO:tensorflow:global step 99: loss = 7.7450 (1.360 sec/step)
INFO:tensorflow:global step 100: loss = 7.1925 (1.355 sec/step)
INFO:tensorflow:global step 101: loss = 7.0041 (1.387 sec/step)
INFO:tensorflow:global step 102: loss = 7.1004 (1.345 sec/step)
INFO:tensorflow:global step 103: loss = 7.2309 (1.350 sec/step)
INFO:tensorflow:global step 104: loss = 6.8801 (1.351 sec/step)
INFO:tensorflow:global step 105: loss = 6.6734 (1.370 sec/step)
INFO:tensorflow:global step 106: loss = 7.2226 (1.334 sec/step)
INFO:tensorflow:global step 107: loss = 6.5237 (1.367 sec/step)
INFO:tensorflow:global step 108: loss = 6.9466 (1.388 sec/step)
INFO:tensorflow:global step 109: loss = 7.1195 (1.341 sec/step)
INFO:tensorflow:global step 110: loss = 7.3159 (1.378 sec/step)
INFO:tensorflow:global step 111: loss = 6.9681 (1.327 sec/step)
INFO:tensorflow:global step 112: loss = 6.7221 (1.372 sec/step)
INFO:tensorflow:global step 113: loss = 7.4709 (1.322 sec/step)
INFO:tensorflow:global step 114: loss = 6.3237 (1.404 sec/step)
INFO:tensorflow:global step 115: loss = 7.1912 (1.372 sec/step)
INFO:tensorflow:global step 116: loss = 6.7614 (1.382 sec/step)
INFO:tensorflow:global step 117: loss = 7.3980 (1.353 sec/step)
INFO:tensorflow:global step 118: loss = 7.2393 (1.389 sec/step)
INFO:tensorflow:global step 119: loss = 7.0953 (1.344 sec/step)
INFO:tensorflow:global step 120: loss = 6.9358 (1.340 sec/step)
INFO:tensorflow:global step 121: loss = 7.1051 (1.359 sec/step)
INFO:tensorflow:global step 122: loss = 6.9368 (1.380 sec/step)
INFO:tensorflow:global step 123: loss = 6.7352 (1.376 sec/step)
INFO:tensorflow:global step 124: loss = 7.1106 (1.349 sec/step)
INFO:tensorflow:global step 125: loss = 6.8234 (1.339 sec/step)
INFO:tensorflow:global step 126: loss = 6.5375 (1.383 sec/step)
INFO:tensorflow:global step 127: loss = 6.4937 (1.353 sec/step)
INFO:tensorflow:global step 128: loss = 6.7102 (1.365 sec/step)
INFO:tensorflow:global step 129: loss = 6.9827 (1.339 sec/step)
INFO:tensorflow:global step 130: loss = 7.1660 (1.334 sec/step)
INFO:tensorflow:global step 131: loss = 6.6511 (1.348 sec/step)
INFO:tensorflow:global step 132: loss = 6.7159 (1.352 sec/step)
INFO:tensorflow:global step 133: loss = 6.7683 (1.344 sec/step)
INFO:tensorflow:global step 134: loss = 7.1323 (1.352 sec/step)
INFO:tensorflow:global step 135: loss = 6.2052 (1.377 sec/step)
INFO:tensorflow:global step 136: loss = 6.6035 (1.370 sec/step)
INFO:tensorflow:global step 137: loss = 6.6624 (1.361 sec/step)
INFO:tensorflow:global step 138: loss = 6.6252 (1.414 sec/step)
INFO:tensorflow:global step 139: loss = 6.9732 (1.358 sec/step)
INFO:tensorflow:global step 140: loss = 6.7813 (1.342 sec/step)
INFO:tensorflow:global step 141: loss = 6.6474 (1.368 sec/step)
INFO:tensorflow:global step 142: loss = 6.5325 (1.328 sec/step)
INFO:tensorflow:global step 143: loss = 6.7729 (1.347 sec/step)
INFO:tensorflow:global step 144: loss = 7.0516 (1.360 sec/step)
INFO:tensorflow:global step 145: loss = 6.6783 (1.378 sec/step)
INFO:tensorflow:global step 146: loss = 6.4639 (1.388 sec/step)
INFO:tensorflow:Recording summary at step 146.
INFO:tensorflow:global_step/sec: 0.72413`
